File: c++/latihan/Pendahuluan Kelas dan Relasinya/latihan3/titik.cpp
#include <iostream>
#include <string>

using namespace std;

class Titik{
    private:
        int x,y;
    public:
        Titik(){
            x = 0;
            y = 0;
        }
        Titik(int xp,int yp){
            x = xp;
            y = yp;
        }

        void setx(int xp){
            x = xp;
        }

        void sety(int yp){
            y = yp;
        }

        int getx(){
            return x;
        }

        int gety(){
            return y;
        }

        ~Titik(){ 
        }
};

File: php/latihan/Pendahuluan Kelas dan Relasinya/latihan3/titik.php
<?php
    class Titik{
        private $x;
        private $y;

        public function __construct(){
            $this->x = 0;
            $this->y = 0;
        }

        public function setx($xp){
            $this->x = $xp;
        }

        public function sety($yp){
            $this->y = $yp;
        }

        public function getx(){
            return $this->x;
        }

        public function gety(){
            return $this->y;
        }

        function __destruct(){
            
        }
    }
?>